packages/Popover/src/index.d.ts

```diff
@@ -53,6 +53,8 @@ declare namespace Popover {
     [x: string]: any;
 
     children?: React.ReactNode;
+
+    isAlwaysOnDom?: boolean;
```

Comment:
Do we consider props like `shouldUnmount` ? btw I think `Dom` should be all caps like `DOM`